apps/web-mzima-client/src/app/shared/components/fragments/share-and-donate/share-and-donate.component.html

```diff
@@ -20,15 +20,20 @@
         class="menu__footer__button"
         *ngIf="isDonateAvailable"
       ></app-donation-button>
-      <mzima-client-button
+
+      <!-- ------------- -->
+      <!-- Removed share popup so that user can easily navigate pages on the mobile screen or smaller screen -->
+      <!-- -------- -->
+
+      <!-- <mzima-client-button
         fill="outline"
         color="transparent-style-1"
         (buttonClick)="openShare()"
         class="menu__footer__button"
       >
         {{ 'app.share' | translate }}
         <mat-icon icon svgIcon="share"></mat-icon>
-      </mzima-client-button>
+      </mzima-client-button> -->
     </div>
```

Comment:
If you want to remove a piece of code, always delete it completely instead of commenting it out :) It makes the code easier to read and if we need to see the history, we can always use the git-history :)

Reply:
yeah, thanks for giving me the hint ..I just fixed that issue by adding the bottom position of 125px ..now it perfectly works..thanks again for pointing that..